test/e2e/node/examples.go

```diff
@@ -53,17 +53,17 @@ var _ = framework.KubeDescribe("[Feature:Example]", func() {
 
 		// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
 		// lying around so we don't have to race any caches
-		err := auth.BindClusterRoleInNamespace(c.RbacV1(), "edit", f.Namespace.Name,
+		err := e2eauth.BindClusterRoleInNamespace(c.RbacV1(), "edit", f.Namespace.Name,
 			rbacv1.Subject{Kind: rbacv1.ServiceAccountKind, Namespace: f.Namespace.Name, Name: "default"})
 		framework.ExpectNoError(err)
 
-		err = auth.WaitForAuthorizationUpdate(c.AuthorizationV1(),
+		err = e2eauth.WaitForAuthorizationUpdate(c.AuthorizationV1(),
 			serviceaccount.MakeUsername(f.Namespace.Name, "default"),
 			f.Namespace.Name, "create", schema.GroupResource{Resource: "pods"}, true)
 		framework.ExpectNoError(err)
 	})
 
-	framework.KubeDescribe("Liveness", func() {
+	SIGDescribe("Liveness", func() {
```

Comment:
SIGDescribe shouldn't be nested, this can be`ginkgo.Describe` (you'll need to import); `[sig-node] [Feature:Example] [sig-node] [Liveness]` seems a bit redundant